python-requirements.txt

```diff
@@ -2,6 +2,8 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+hjson
```

Comment:
not needed any more?

Reply:
hjson forever!

Well spotted, I shall remove it.